packages/create-vite/package.json

```diff
@@ -13,7 +13,11 @@
     "template-*",
     "dist"
   ],
-  "main": "index.js",
+  "exports": {
+    ".": {
+      "import": "./index.js"
+    }
+  },
```

Comment:
This is technically a breaking change though, but also I don't see why we need to export anything for a CLI package. Maybe we can remove `main` directly?